tests/kafkatest/services/console_consumer.py

```diff
@@ -273,8 +276,25 @@ def _worker(self, idx, node):
         with self.lock:
             self.read_jmx_output(idx, node)
 
+    def _wait_until_partitions_assigned(self, node, timeout_sec=60):
+        if self.jmx_object_names is not None:
+            raise Exception("'wait_until_partitions_assigned' is not supported while using 'jmx_object_names'/'jmx_attributes'")
```

Comment:
this is not ideal but requires more refactoring to be able support multiple JmxTools running at once